07-flex-layout-2: Add disclaimer to Desired Outcome (#481)
TheOdinProject/css-exercises: flex/07-flex-layout-2/README.md

![smaller](./desired-outcome-smaller.png)

> Note: The emojis may instead show up as one or several text symbols (e.g. &#9734;&#9794;) if the Noto Color Emoji font family is not properly installed. This does not affect the exercise and can be ignored.

Reviewer: Just to make it a little less specific about that particular font family, what do you think about:

```suggestion
Note: The emojis may instead show up as one or several text symbols (e.g. &#9734;&#9794; if you don't have an emoji based font family installed on your operating system. This does not affect the exercise and can be ignored.
```

Author: Makes sense, I've included the change - thanks for reviewing!
